### PR TITLE
Skip test_lstm_simple_precommit on gpu

### PR DESCRIPTION
**Pull request by just-sparta**
### Tickets:
 - *56534*

### CODE REVIEW

**Comment by just-sparta** [comment]
due to 56534

**Review by dkrupnov — APPROVED** [review]

**Comment by just-sparta** [comment]
@alexander-shchepetov, @lazarevevgeny please review

File: tests/layer_tests/onnx_tests/test_lstm.py [hunk 1]
```diff
@@ -143,6 +143,9 @@ def create_lstm(self, direction: str, cell_type: str, hidden_size=128):
     @pytest.mark.parametrize('direction', ["forward", "bidirectional", "reverse"])
     @pytest.mark.parametrize('cell_type', ["LSTM", "GRU", "RNN"])
     def test_lstm_simple_precommit(self, direction, cell_type, ie_device, precision, ir_version, temp_dir):
+        if ie_device == 'GPU':
+            # TODO: add test to precommit after heavy/light optimization
+            pytest.skip("Sporadic TimeoutError (Inference Engine running timed out) on GPU")
```
Comment (lazarevevgeny) [inline tests/layer_tests/onnx_tests/test_lstm.py:148]: We usually add a ticket number in the code so we know how to track the progress of the activity. Please, create the ticket to GPU plugin and leave a reference here.

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: tests/layer_tests/onnx_tests/test_lstm.py (hunk 1 above)
Comment (just-sparta) [reply tests/layer_tests/onnx_tests/test_lstm.py:148]: I think that this failed test is a consequence of the use of parallelization. In the near future, work will be carried out to divide the scope of tests into heavy and light models (CVS-55862), with different numbers of workers for parallel execution.
After this work, the test will be returned to the scope.

File: tests/layer_tests/onnx_tests/test_lstm.py [hunk 2]
```diff
@@ -140,6 +140,7 @@ def create_lstm(self, direction: str, cell_type: str, hidden_size=128):
         return onnx_net, None
 
     @pytest.mark.precommit
+    @pytest.mark.timeout(300)  # "Sporadic TimeoutError (Inference Engine running timed out)"
```
Comment (vurusovs) [inline tests/layer_tests/onnx_tests/test_lstm.py:143]: This change won't fix the issue due inference has it's own timeout that should be increased for the model (see https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/common/layer_utils.py#L29)

Comment (vurusovs) [inline tests/layer_tests/onnx_tests/test_lstm.py:143]: This change won't fix the issue due inference has it's own timeout that should be increased for the model (see https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/common/layer_utils.py#L29)

**Review by vurusovs — CHANGES_REQUESTED** [review]

File: tests/layer_tests/common/layer_test_class.py [hunk 3]
```diff
@@ -24,7 +24,7 @@ def produce_model_path(self, framework_model, save_path):
     def get_framework_results(self, inputs_dict, model_path):
         pass
 
-    def _test(self, framework_model, ref_net, ie_device, precision, ir_version, temp_dir,
+    def _test(self, framework_model, ref_net, ie_device, precision, ir_version, temp_dir, timeout=60,
```
Comment (vurusovs) [inline tests/layer_tests/common/layer_test_class.py:27]: please, rename to `infer_timeout` because it's not a full test timeout

Comment (vurusovs) [inline tests/layer_tests/common/layer_test_class.py:27]: please, rename to infer_timeout because it's not a full test timeout

**Review by vurusovs — CHANGES_REQUESTED** [review]

**Review by vurusovs — APPROVED** [review]

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**